lib/src/pages/credits.dart

```diff
@@ -0,0 +1,81 @@
+import "package:flutter/material.dart";
+
+import "package:ramaz/widgets.dart";
+import "package:ramaz/data.dart";
+
+import "drawer.dart";
+
+/// Creates the Credits Page by iterating through each ContributorCard.
+class CreditsPage extends StatelessWidget {
+	/// Builds the Credits page.
+	const CreditsPage();
+
+	@override 
+	Widget build (BuildContext context) => ResponsiveScaffold(
+		drawer: const NavigationDrawer(),
+		appBar: AppBar(title: const Text ("Credits")),
+		bodyBuilder: (_) => ListView(
+			children: [
+				for (final Contributor contributor in Contributor.contributors)
+					ContributorCard(contributor)
+			]
+		)
+	);
+}
+/// A class that creates each individual Contributor's Card.
+/// It creates a Row that holds the Image, and then a Column that
+/// holds the rest of the information.
```

Comment:
```suggestion
/// A class that creates each individual Contributor's Card.
/// 
/// It creates a Row that holds the Image, and then a Column that
/// holds the rest of the information.
```

Make sure you leave a blank line with `///`. Dartdoc will consider the first line a "title" or short description and will show it separately from the rest of the documentation.